module/Application/view/error/404.phtml

```diff
@@ -1,106 +1,102 @@
-<h1><?php echo $this->translate('A 404 error occurred') ?></h1>
-<h2><?php echo $this->message ?></h2>
+<h1>A 404 error occurred</h1>
+<h2><?= $this->message ?></h2>
 
-<?php if (isset($this->reason) && $this->reason): ?>
+<?php if (isset($this->reason) && $this->reason) : ?>
```

Comment:
We can use just `!empty($this->reason)` here instead of `isset($this->reason) && $this->reason`.